### PR TITLE
Fix crafting table open error

### PR DESCRIPTION
**Pull request by EntryPointKR**
When open the crafting table, current PS will send a inv title exactly "Unknown translation key: block.minecraft.crafting_table.name" to 1.4-1.7
Then 1.4~1.7 will validate the title size, crashes.
Refer issue: #889

### CODE REVIEW

**Comment by Shevchik** [comment]
Translation files needs to be updated to actually translate that key instead of hacking around. Name clamping is still needed tho.

**Comment by EntryPointKR** [comment]
Thanks for review

**Comment by EntryPointKR** [comment]
Oh wait. Does "Only clamp the name" mean like that?
The workbench title is not required in 1.4~1.7 so I replaced to empty string

File: src/protocolsupport/protocol/packet/middle/clientbound/play/MiddleInventoryOpen.java [hunk 1]
```diff
@@ -45,4 +46,7 @@ public boolean postFromServerRead() {
 		}
 	}
 
+	protected String getLegacyTitle() {
```
Comment (Shevchik) [inline src/protocolsupport/protocol/packet/middle/clientbound/play/MiddleInventoryOpen.java]: inline to callers

**Pull request merged**